  server: evarobotDSK
  portNo: 9090
  topicBumper: bumper
  frequency: 10.0
  alwaysOn: true
